pom.xml

```diff
@@ -394,6 +394,36 @@
                 <version>${slf4j.api.version}</version>
                 <scope>test</scope>
             </dependency>
+
```

Comment:
Looks like org.jacoco.agent, testng and slf4j-api dependencies are already there in the pom. Remove the duplicates.